app/helpers/lookup_keys_helper.rb

```diff
@@ -118,7 +118,7 @@ def lookup_key_warnings(required, has_value)
       { :text => _("Required parameter without value.<br/><b>Please override!</b><br/>"),
         :icon => "error-circle-o" }
     else
-      { :text => _("Optional parameter without value.<br/><i>Will not be sent to Puppet.</i><br/>"),
+      { :text => _("Optional parameter without value.<br/><i>Still managed by Foreman, the value will be empty.</i><br/>"),
```

Comment:
This change is not quite clear and the meaning seems to be different then before - empty value sent instead of not sent at all?